launcher/test-framework/org.jboss.tools.common.launcher.reddeer/src/org/jboss/tools/common/launcher/reddeer/wizards/NewLauncherProjectWizard.java

```diff
@@ -0,0 +1,60 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributor:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.common.launcher.reddeer.wizards;
+
+
+import org.eclipse.jface.wizard.Wizard;
+import org.eclipse.reddeer.common.wait.TimePeriod;
+import org.eclipse.reddeer.common.wait.WaitUntil;
+import org.eclipse.reddeer.swt.impl.browser.InternalBrowser;
+import org.eclipse.reddeer.swt.impl.button.NextButton;
+import org.eclipse.reddeer.swt.impl.shell.DefaultShell;
+import org.eclipse.reddeer.swt.impl.tree.DefaultTreeItem;
+import org.eclipse.reddeer.workbench.impl.shell.WorkbenchShell;
+import org.jboss.tools.central.reddeer.api.JavaScriptHelper;
+import org.jboss.tools.central.reddeer.wait.CentralIsLoaded;
+import org.eclipse.reddeer.eclipse.ui.dialogs.NewWizard;
+
+
+/**
+ * 
+ * @author zcervink
+ *
+ */
+public class NewLauncherProjectWizard extends Wizard {
```

Comment:
I am afraid that you are extending eclipse.jface wizard class instead of reddeer's wizard dialog. Please use org.eclipse.reddeer.jface.wizard.WizardDialog that is tailored to handle dialogs that hold wizard pages objects. Also, this dialog takes care of button like Next, Finish, etc... These methods (finish) should be implemented here. Please, see how the classes are implemented ([QuickFixWizard](https://github.com/eclipse/reddeer/blob/37f27affd709a9622d84e8d29d047dc78a0de1b4/plugins/org.eclipse.reddeer.eclipse/src/org/eclipse/reddeer/eclipse/ui/views/markers/QuickFixWizard.java) and [QuickFixPage](https://github.com/eclipse/reddeer/blob/37f27affd709a9622d84e8d29d047dc78a0de1b4/plugins/org.eclipse.reddeer.eclipse/src/org/eclipse/reddeer/eclipse/ui/views/markers/QuickFixPage.java)) and check the usage https://github.com/jbosstools/jbosstools-integration-tests/blob/a6805629f04eff5dff33de118fc11bee01fd9425/tests/org.jboss.tools.cdi.bot.test/src/org/jboss/tools/cdi/bot/test/beansxml/validation/template/BeansXMLValidationQuickFixTemplate.java#L214.